### DISCUSSION TITLE
Any experience with Danfoss Ally Room Sensor?

### DISCUSSION THREAD

**Discussion by michaelsidenius** [Koenkk/zigbee2mqtt#14046] [Devices]
Thinking about investing in those (despite their hefty price). Should be Zigbee 3 compliant. Danfoss is generaly high quality. Anyone tried them with Z2M? [Danfoss](https://www.danfoss.com/en/about-danfoss/news/dhs/the-newest-addition-to-the-danfoss-ally-family/)

**Comment by stachjankowski** [comment]
I'm trying to use them at home and unfortunately after some time (a few hours) they stop receiving messages from the coordinator.
I bought them because of the Danfoss brand.
My coordinator: Sonoff ZBDongle-P
The problem is quite well described here https://github.com/Koenkk/zigbee2mqtt/issues/13478
Currently looking for a solution to my problem.
